content/installation/net/install/WebAPI-Owin-Installation.md

```diff
@@ -0,0 +1,94 @@
+## WebAPI hosted by OWIN deployed as a Windows service
+
+Install the service that contains the WebAPI application. Note the name of the service.
+
+Set the required environment variables. Environment variables that are unique for each service can be set under that service's registry key. The service's registry key can be found at:  
+HKLM\SYSTEM\CurrentControlSet\Services\YourServiceName
+
+Under the service's registry key, create a REG_MULTI_SZ value called `Environment`. Then add the environment variables and values below. Note that each environment variable key/value pair must be separated by a new line. If there is already an `Environment` value, add the new values below the existing values.
+
```

Comment:
We may want to note that the PROFILER_PATH environment variables should match the installation directory chosen for the .NET Framework agent.